### PR TITLE
Fb1315

### PR DESCRIPTION
**Pull request by spiresheep**

### CODE REVIEW

File: Applications/WebPortal/WebApp/library/source/pages/loading_page.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { Transition } from 'react-transition-group';
+import { HBoxLayout, Padding, VBoxLayout } from '..';
+
+interface Properties {
+
+  /** Ammount of time in milliseconds before
```
Comment (kranar) [inline Applications/WebPortal/WebApp/library/source/pages/loading_page.tsx]: Typo on ```amount```.

File: Applications/WebPortal/WebApp/library/source/pages/loading_page.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+interface State {
+  showAnimation: boolean;
+}
+
+/** Displays a loading page. */
+export class LoadingPage extends React.Component<Properties, State> {
+  public static readonly defaultProps = {
+    delay: 0
```
Comment (kranar) [inline Applications/WebPortal/WebApp/library/source/pages/loading_page.tsx]: The delay in the spec is something like 1 or 2 seconds.

File: Applications/WebPortal/WebApp/library/source/pages/loading_page.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+              </div>)}
+          </Transition>
+          <Padding/>
+        </HBoxLayout>
+        <Padding size={LoadingPage.BOTTOM_PADDING}/>
+      </VBoxLayout>);
+  }
+
+  public componentDidMount(): void {
+    this.setState({showAnimation: true});
```
Comment (kranar) [inline Applications/WebPortal/WebApp/library/source/pages/loading_page.tsx]: Does this state do anything? What happens if you remove this state and pass a value of ```true``` to the ```Transition```'s ```in``` prop?

Comment (spiresheep) [reply Applications/WebPortal/WebApp/library/source/pages/loading_page.tsx]: I originally tried it with the `in` prop being set to `true` and the transition would not occur.

Comment (kranar) [reply Applications/WebPortal/WebApp/library/source/pages/loading_page.tsx]: Okay, keep it as is then.

**Pull request merged**